src/hooks/useAuthInfo.ts

```diff
@@ -13,6 +13,7 @@ export type UseAuthInfoLoading = {
     isImpersonating: undefined
     impersonatorUserId: undefined
     refreshAuthInfo: () => Promise<void>
+    getAccessTokenForOrg: (orgId: string) => Promise<AccessTokenForActiveOrg>
```

Comment:
Could you make a `tokens` object which has this function on it? We'll put any other relevant functions we make down the road on it as well.